### ISSUE TITLE
Pylance incorrectly report accessibility of some code

### ISSUE THREAD

**Issue by banksempire** [microsoft/pylance-release#5675]
## Environment data

-   Language Server version: v2024.3.1
-   OS and version: MacOS 14.4
-   Python version (& distribution if applicable, e.g. Anaconda): Anaconda Python 3.11.8

## Code Snippet

```python
import pandas as pd

pd.concat([])

1 + 1 # pylance thinks this line of code is inaccessible
```

## Repro Steps

1. Copy the code into a .py file

## Expected behavior

pylance should report line "1 + 1" is accessible.

## Actual behavior

pylance reported line "1 + 1" is inaccessible.
<img width="455" alt="image" src="https://github.com/microsoft/pylance-release/assets/12817392/4e7335ce-37ac-4081-910c-021e9eb9db1f">
## Logs

```
XXX
```

**Comment by debonte** [comment]
This is a duplicate of #5630. The [underlying issue](https://github.com/pandas-dev/pandas-stubs/issues/888) was in `pandas-stubs`. They have fixed it and their fix will ship in our next prerelease build, likely today.

In the meantime there's a workaround described here: https://github.com/microsoft/pylance-release/issues/5630#issuecomment-1994616891

**Comment by debonte** [comment]
This issue is also fixed in our most recent stable release -- [2024.3.2](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202432-25-March-2024-Release)

**Comment by banksempire** [comment]
Much appreciated!
